pandas/tests/test_nanops.py

```diff
@@ -994,6 +994,49 @@ def prng(self):
         return np.random.RandomState(1234)
 
 
+@pytest.fixture(params=[
+    pd.Series([1, 2, 3, 4, 5, 6]),
+    pd.DataFrame([[1, 2, 3], [4, 5, 6]])
+])
+def nan_test_object(request):
+    return request.param
+
+
+@pytest.mark.parametrize("standard, nan_method", [
+    (np.sum, np.nansum),
+    (np.mean, np.nanmean),
+    (np.std, np.nanstd),
+    (np.var, np.nanvar),
+    (np.median, np.nanmedian),
+    (np.max, np.nanmax),
+    (np.min, np.nanmin),
+])
+def test_np_nan_functions(standard, nan_method, nan_test_object):
+    tm.assert_almost_equal(nan_test_object.agg(standard),
```

Comment:
Might be a silly question but can we not use the frame / series equals methods here? Don't think precision is that much of a factor with the fixtures

Reply:
I think these are ok as ``nan_test_object`` can be either a series or a frame. I think the name should be changed though, to e.g. ``series_or_frame``.